src/undomanager.js

```diff
@@ -1,12 +1,14 @@
 define([
     'jquery',
     'underscore',
+    'vellum/util',
     'tpl!vellum/templates/undo_alert',
 ], function(
     $,
     _,
-    UNDO_ALERT
-) {
+    util,
+    UNDO_ALERT,
+undefined) {
```

Comment:
I don't think there's a good way to compromise on aesthetics, so I'll defer to the current vellum conventions & drop it ;)